### PR TITLE
Bumped @nexes/mongo-knex to version 0.1.1

### PR DESCRIPTION
**Pull request by kirrg001**
refs #1

- mongo-knex supports some relation cases already
- forward options to mongo-knex
- remove tests which have a wrong assertions
  - these tests are removed in https://github.com/NexesJS/NQL/pull/5 anyway

### CODE REVIEW

**Comment by kirrg001** [comment]
@gargol Can i 🖕 ?

File: lib/nql.js [hunk 1]
```diff
@@ -33,7 +33,7 @@ module.exports = (queryString, options) => {
     };
 
     // Use MongoKnex to apply the query to a query builder object
-    api.querySQL = qb => mongoKnex(qb, api.parse());
+    api.querySQL = qb => mongoKnex(qb, api.parse(), options);
```
Comment (gargol) [inline lib/nql.js]: `options` - this is where `filters` and `relations config` would come through?

Comment (kirrg001) [reply lib/nql.js]: yes

Comment (kirrg001) [reply lib/nql.js]: not filters, only the config

Comment (gargol) [reply lib/nql.js]: How would filters come in then?

Comment (kirrg001) [reply lib/nql.js]: https://github.com/NexesJS/NQL/blob/master/lib/nql.js#L13

Comment (kirrg001) [reply lib/nql.js]: NQL supports a single filter currently.

Comment (kirrg001) [reply lib/nql.js]: The PR is about: bumping mongo-knex and forwarding the options. The options are needed to use relation queries.

Comment (gargol) [reply lib/nql.js]: Sure, makes sense. Just asking :) :shipit:

**Pull request merged**